src/App.vue

```diff
@@ -753,6 +753,7 @@ export default {
         businessIdentifier: this.getIdentifier,
         commentsCount: application.commentsCount,
         commentsLink: application.commentsLink,
+        displayLedger: application.displayLedger,
```

Comment:
This needs to be explicitly copied from the application object to the filing item object.

This code is a work-around for displaying "bootstrap filings" like regular filings. One day, it would be nice for these filings to come consistently from the "tasks" or "filings" endpoints...

Example single bootstrap filing from "filings" endpoint:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/1a0831b0-e8bc-4bd7-b2b7-decb8e5aade5)

Compare with list of filings from "filings" endpoint:

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/38e60bbd-482b-4d74-a32b-5f830fda6b36)